app/src/main/res/values/dimens.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
     <dimen name="dimension_4dp">4dp</dimen>
     <dimen name="dimension_8dp">8dp</dimen>
+    <dimen name="dimension_10dp">10dp</dimen>
```

Comment:
Unused, can be deleted.

Reply:
I am using the 4dp and 8dp one for the IntroScreen. I will delete the 10dp  in the next commit